core/src/main/java/org/apache/accumulo/core/conf/AccumuloConfiguration.java

```diff
@@ -330,6 +330,23 @@ public int getCount(Property property) {
     return Integer.parseInt(countString);
   }
 
+  /**
+   * Gets the default value of a property of type {@link PropertyType#COUNT}, interpreting the value
+   * properly (as an integer).
+   *
+   * @param property
+   *          property to get default value
+   * @return property default value
+   * @throws IllegalArgumentException
+   *           if the property is of the wrong type
+   */
+  public int getDefaultCount(Property property) {
```

Comment:
This doesn't appear to be used anywhere

Reply:
Yeah this was holdover from an earlier commit.

Reply:
I don't think its needed anymore but I was waiting to get feedback from @ctubbsii

Reply:
It should be removed. It is equivalent to, and redundant with, `DefaultConfiguration.getInstance().getCount(property)`.